Detections/SecurityEvent/SecurityEventLogCleared.yaml

```diff
@@ -17,9 +18,9 @@ relevantTechniques:
   - T1107
 query: |
 
-  let timeframe = 1d;
+ let timeframe = 1d;
```

Comment:
Something I did not notice before, we lost a space here, this breaks YAML, please add the additional space in front of let.  You can verify format with any online YAML validator, I often use - https://codebeautify.org/yaml-validator/

Reply:
Also resolved this.